test/java/seedu/addressbook/parser/UtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.parser;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import seedu.addressbook.common.Utils;
+
+import java.util.ArrayList;
+
+public class UtilsTest {
+
+    @Test
+    public void testIsAnyNull() {
```

Comment:
the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.